### ISSUE TITLE
UI for forwarding messages has completely disappeared

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-web#14690]
although it still works, it doesn't prompt you to select the destination whatsoever. needs design urgently

**Comment by turt2live** [comment]
backstory: years of regressions and lack of attention have slowly dwindled this down to just an `h2` hovering near the top of the page. It used to have all kinds of neat fading effects, actual backgrounds, clear instructions, etc.

**Comment by t3chguy** [comment]
Related https://github.com/vector-im/riot-web/issues/5641

**Comment by t3chguy** [comment]
Related https://github.com/vector-im/riot-web/issues/11195

**Comment by jryans** [comment]
I think this is a duplicate of https://github.com/vector-im/riot-web/issues/10165. (Also, it seems unrelated to new-room-list?)

**Comment by t3chguy** [comment]
I mean you forward by clicking the room list but yes otherwise

I would say that https://github.com/vector-im/riot-web/issues/5641 covers the whole situation well enough, its just poor UX, now more than ever

**Comment by niquewoodhouse** [comment]
Quick wireframe of this using the notification settings modal. A bit rushed but just to aid discussion as I don't know what the UI used to look like.

- Can preview message for context and re-assurance
- Can't add a message
- Can multi-pick rooms/chats

<img width="1151" alt="01  fwd--msg--simple" src="https://user-images.githubusercontent.com/1587166/88398387-3df1ef00-cdbd-11ea-89d0-5332fbe424a5.png">


Is this roughly what it was like? Does it seem feasible and good?

**Comment by t3chguy** [comment]
Previously (completely developer design) it gave you a modal-like prompt to select a room in the Room List, and upon you clicking that room in the room list it'd forward the message there.

**Comment by niquewoodhouse** [comment]
[Here's](https://www.figma.com/file/rHnynEgrePUtXVear8rGT0/34-Room-List?node-id=2453%3A141308) a UI that I think makes sense.  It's not pixel perfect, just an indication of what would make sense from a design perspective.


**1. Fwd message**
<img width="1151" alt="fwd--msg@2x" src="https://user-images.githubusercontent.com/1587166/88551937-5c552600-d01b-11ea-8b9e-457ad9053890.png">
- Preview of message (uses same component as reply in timeline)


**2. Sending**
<img width="1151" alt="fwd--msg-feedback@2x" src="https://user-images.githubusercontent.com/1587166/88552091-91617880-d01b-11ea-8233-76812bb0eba0.png">


**3.  Forwarded message**
<img width="1151" alt="01  fwd--msg--simple@2x" src="https://user-images.githubusercontent.com/1587166/88554034-e69e8980-d01d-11ea-8570-482d18d59807.png">

In testing this, it doesn't look like forwarded messages include the name/image of the original author or where's it's from, which I think isn't ideal.  So I'd advocate for it to look like the above.

**Comment by t3chguy** [comment]
> In testing this, it doesn't look like forwarded messages include the name/image of the original author or where's it's from, which I think isn't ideal. So I'd advocate for it to look like the above.

That would be blocked on a Matrix spec proposal

**Comment by MurzNN** [comment]
Also message forwarding (one or several) can be implemented using message attachment, proposed here: https://github.com/matrix-org/matrix-doc/issues/2289
